DataFormats/GEMRecHit/interface/GEMCSCSegment.h

```diff
@@ -63,13 +61,7 @@ class GEMCSCSegment final : public RecSegment {
 
   int degreesOfFreedom() const override { return 2 * nRecHits() - 4; }
 
-  int nRecHits() const {
-    return (theGEMRecHits.size() + theCSCSegment.specificRecHits().size());
-    edm::LogVerbatim("GEMCSCSegment") << "[GEMCSCSegment :: nRecHits] CSC RecHits: "
```

Comment:
This is the only LogSomething previously coded in a header file which is not replicated here with a function into the updated file: just pointing it out here in case it was forgotten. 
If it was done on purpose, instead, I must agree that this is actually a rather trivial and probably not useful Log message, indeed...

Reply:
This was done on purpose since the log message could never be called as the line before it is a `return`.